### DISCUSSION TITLE
List Bun with JS runtimes

### DISCUSSION THREAD

**Discussion by markgoho** [mdn/mdn-community#313] [Browser compatibility data] [answered]
Currently, MDN lists two server runtimes (Node and Deno) 
![image](https://user-images.githubusercontent.com/9759954/210575438-b42d2b4d-6ea9-4e6d-9ca9-b13510a30a0d.png)

As [Bun](https://bun.sh) grows in popularity and reaches feature parity with these two older runtimes, when does it make sense to start including it? What integrations are required to pull the information in automatically?

**Comment by wbamberg** [comment, marked as answer]
There is a guideline for this! https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines/browsers.md#addition-of-browsers .

**Comment by pepelsbey** [comment]
FYI, there’s a [draft PR](https://github.com/mdn/browser-compat-data/pull/23440) adding Bun to BCD you might want to follow.

Closing this discussion as resolved. Thanks!
